AlfrescoKit/AlfrescoKit/Model/AKNetworkActivity.h

```diff
@@ -0,0 +1,35 @@
+/*
+ ******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile SDK.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *****************************************************************************
+ */
+
+/** AKNetworkActivity protocol
+ 
+ Author: Tauseef Mughal (Alfresco)
+ */
+
+#import <Foundation/Foundation.h>
+
+@class AlfrescoRequest;
+
+@protocol AKNetworkActivity <NSObject>
```

Comment:
Made this mandatory as it encourages users to provide some sort of activity indication to users (rather than just a plain UI). Making it mandatory throws up warnings in Xcode when it hasn't been implemented.